js/each/each.md

```diff
@@ -1,18 +1,21 @@
 @module {function} can-util/js/each/each each
 @parent can-util/js
 
-@signature `each(elements, callback, context)`
+@signature `each(collection, callback, context)`
 
-Loop over each element in an Array-Like data structure.
+Loop over each element in an Array-Like or Object data structure.
 
-  @param {Object|ArrayLike} elements
-  @param {function(element, key, elements)} callback
+  @param {Object|ArrayLike} collection
+  @param {function(element, key, collection)} callback
   @param {Object} [context] the context object
 
-  @return {ArrayLike}  the orignal array of elements
+  @return {collection}  the orignal collection
```

Comment:
I wonder if we should either update the code to use `collection` instead of `elements`, or leave the argument documented as `elements`.